src/Strategy/GithubStrategy.php

```diff
@@ -68,7 +68,7 @@ public function download(Updater $updater)
     {
         /** Switch remote request errors to HttpRequestExceptions */
         set_error_handler(array($updater, 'throwHttpRequestException'));
-        $result = humbug_get_contents($this->remoteUrl);
+        $result = \Humbug\get_contents($this->remoteUrl);
```

Comment:
could you import it? `use function Humbug\get_contents()`, same for the other places